src/components/CodeWithCopy.js

```diff
@@ -28,18 +29,23 @@ const Wrapper = styled.span`
   }
 `
 
-const Base = ({ copy, text, ...props }) => {
-  const getCopyText = () => (copy || text || '').trim()
+const copyIt = (copy, text) => {
+  const copyText = (copy || text || '').trim()
+
+  copyToClipboard(copyText)
+}
+
+const Base = ({ copy, text, color, ...props }) => {
   return (
-    <Wrapper className='code-with-copy'>
+    <Wrapper onClick={stopEvent} {...props} className='code-with-copy'>
```

Comment:
Would it make sense to move the `stopEvent()` a bit deeper, so that clicking the text still works like a click?